zstacklib/zstacklib/utils/lvm.py

```
@@ -13,7 +13,7 @@
 
 import simplejson
 
-from zstacklib.utils import form, report
+from zstacklib.utils import form, report, jsonobject
```

Ruff (0.11.9)

`zstacklib.utils.jsonobject` imported but unused

Remove unused import: `zstacklib.utils.jsonobject`

(F401)